src/Meshes/mesh.ts

```diff
@@ -3486,11 +3514,11 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
         // Material
         if (this.material) {
             if (!this.material.doNotSerialize) {
-                serializationObject.materialId = this.material.id;
+                serializationObject.materialUniqueId = this.material.uniqueId;

```

Comment:
we cannot change the current structure because old system will not be able to load the file anyway
We need to add the additional info (the uniqueId) and not remove the previous one

Reply:
Ah okay, just to confirm my understanding: we need to be able to load files created with Babylon 5.0 in earlier versions of Babylon?

Reply:
as much as possible yes (and here the fix is easy to keep that downlevel support)